ui/components/DashboardComponent/charts/DashboardMeshModelGraph.js

```diff
@@ -69,44 +69,46 @@ function MeshModelContructs({ classes }) {
   );
 
   return (
-    <Link
-      href="/settings?settingsCategory=Registry"
-      style={{
-        pointerEvents: !CAN(keys.VIEW_REGISTRY.action, keys.VIEW_REGISTRY.subject)
-          ? 'none'
-          : 'auto',
-      }}
-    >
-      <div className={classes.dashboardSection}>
-        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-          <Typography variant="h6" gutterBottom className={classes.link}>
-            Registry
-          </Typography>
+    <UsesSistent>
+      <Link
+        href="/settings?settingsCategory=Registry"
+        style={{
+          pointerEvents: !CAN(keys.VIEW_REGISTRY.action, keys.VIEW_REGISTRY.subject)
+            ? 'none'
+            : 'auto',
+        }}
+      >
+        <div className={classes.dashboardSection}>
+          <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+            <Typography variant="h6" gutterBottom className={classes.link}>
+              Registry
+            </Typography>
 
-          <div onClick={(e) => e.stopPropagation()}>
-            <CustomTextTooltip
-              placement="left"
-              interactive={true}
-              variant="standard"
-              title={`The Meshery Registry is a critical component acting as the central repository for all capabilities known to Meshery. [Learn More](https://docs.meshery.io/concepts/logical/registry)`}
-            >
-              <IconButton disableRipple={true} disableFocusRipple={true}>
-                <InfoOutlined
-                  color={theme.palette.secondary.iconMain}
-                  style={{ ...iconSmall, marginLeft: '0.5rem', cursor: 'pointer' }}
-                  onClick={(e) => {
-                    e.stopPropagation();
-                  }}
-                />
-              </IconButton>
-            </CustomTextTooltip>
+            <div onClick={(e) => e.stopPropagation()}>
+              <CustomTextTooltip
+                placement="left"
+                interactive={true}
+                variant="standard"
+                title={`The Meshery Registry is a critical component acting as the central repository for all capabilities known to Meshery. [Learn More](https://docs.meshery.io/concepts/logical/registry)`}
+              >
+                <IconButton disableRipple={true} disableFocusRipple={true}>
+                  <InfoOutlined
+                    color={theme.palette.secondary.iconMain}
```

Comment:
```suggestion
                    color={theme.palette.icon.default}
```

this is a sistent token.